specs/online_order_spec.rb

```diff
@@ -3,69 +3,137 @@
 require 'minitest/skip_dsl'
 
 # TODO: uncomment the next line once you start wave 3
-# require_relative '../lib/online_order'
+require_relative '../lib/onlineorder'
 # You may also need to require other classes here
 
 # Because an OnlineOrder is a kind of Order, and we've
 # already tested a bunch of functionality on Order,
 # we effectively get all that testing for free! Here we'll
 # only test things that are different.
 
-xdescribe "OnlineOrder" do
+describe "OnlineOrder" do
   describe "#initialize" do
     it "Is a kind of Order" do
-      # Check that an OnlineOrder is in fact a kind of Order
-
+      id = 1
+	  products = {'socks' => 2.34}
+	  customer = 5
+	  status = 'paid'
       # Instatiate your OnlineOrder here
-      # online_order =
-      # online_order.must_be_kind_of Grocery::Order
+      online_order = Grocery::OnlineOrder.new(id, products, customer, status)
+      online_order.must_be_kind_of Grocery::OnlineOrder
```

Comment:
The intention here was to have a line like

```ruby
online_order.must_be_kind_of Grocery::Order
```

Which will verify the inheritance between `OnlineOrder` and `Order`